resources/privacy-and-gdpr.md

```diff
@@ -6,6 +6,251 @@ Before implementing these controls, read the [Adobe Experience Platform Privacy
 
 When Adobe provides software and services to an enterprise, Adobe acts as a data processor for any personal data it processes and stores as part of providing these services. As a data processor, Adobe processes personal data in accordance with your company’s permission and instructions, as set out in your agreement with Adobe. As a data controller, you can use the Experience Platform SDKs to support privacy retrieve and delete requests from your mobile apps.
 
+# Setup steps
+
+The following sections provide details on how you can collect consent and privacy status to ensure collection of data suits your user's preferences.
+
+Depending on the mobile extensions you use, there are two ways of collecting and enforcing consent preferences when using the Experience Platform SDKs as follows:
+
+1. When using the **Edge Network** mobile extensions, you should use the [Consent for Edge Network](../foundation-extensions/consent-for-edge-network) extension.
+2. When using **Adobe Experience Cloud** mobile extensions, you should use privacy status settings.
+
+The two options are documented in detail below.
+
+{% hint style="info" %}
+If you are using a mix of Edge Network and Adobe Experience Cloud mobile extensions, please follow the steps for configuring both consent and privacy status settings.
+{% endhint %}
+
+## Using Experience Platform SDKs for Edge Network
+
+## Update and get collect consent preferences
+
+You can set the collect consent status to ensure collection of data suits your user's preferences.
+
+| Extension        | Collect (y)   | Collect (n)   | Collect (pending) |
+| :--------------- | :------------ | :------------ | :---------------- |
+| **Edge Network** | Hits are sent | Hits not sent | Hits queued       |
+
+> **Note:** When no default collect consent value is defined in configuration, the SDK defaults to Yes \(y\) for collect consent.
```

Comment:
It shouldn't display the backslashes, but I just don't like having random extra characters that are unneeded, especially since it just clutters it up visually when I'm trying to edit.

Especially since there are cases where escape characters _are_ necessary, and having them here makes it a little more confusing for me to edit.